Sources/FodyRunner.UnityIntegration/EditorWeaver.cs

```diff
@@ -51,6 +52,11 @@ private static void WeaveAllAssemblies()
                 if (didChangeAnyAssembly)
                 {
                     AssetDatabase.Refresh();
+                    MethodInfo checkMethod = typeof(EditorUtility).GetMethod("RequestScriptReload", BindingFlags.Public | BindingFlags.Static);
```

Comment:
Then with the alias, this would be updated to:

```
Reflection.MethodInfo checkMethod = typeof(EditorUtility).GetMethod("RequestScriptReload", Reflection.BindingFlags.Public | Reflection.BindingFlags.Static);
```